src/main/java/strategy/ReturnOperationHandler.java

```diff
@@ -0,0 +1,10 @@
+package strategy;
+
+import java.util.Map;
+
+public class ReturnOperationHandler implements OperationHandler {
+    @Override
+    public void apply(Map<String, Integer> inventory, String fruit, int quantity) {
+        inventory.put(fruit, inventory.getOrDefault(fruit, 0) + quantity);
+    }
```

Comment:
Consider verifying the quantity before applying the return operation. According to the checklist, when handling the 'Purchase' operation, we should check the result balance before saving it in the Storage - it should be positive. Although this is the 'Return' operation, it's a good practice to validate inputs to ensure data integrity. Throw a RuntimeException if the quantity is negative.